services/director-v2/tests/unit/test_modules_notifier.py

```diff
@@ -179,4 +179,4 @@ async def test_notifier_publish_message(
                 )
             )
 
-    await _assert_call_count(server_disconnect, call_count=number_of_clients)
+    await _assert_call_count(server_disconnect, call_count=number_of_clients * 2)
```

Comment:
Same as in https://github.com/ITISFoundation/osparc-simcore/pull/7316#discussion_r1984710793

Reply:
already cleared this out